src/helpers/useSelection.ts

```diff
@@ -172,6 +177,10 @@ export const useSelection = ({
     clearSelections(path.map(p => p.id as string));
   };
 
+  const focusOnNode = (item: string) => {
+    ref.current?.centerGraph([item]);
```

Comment:
ref.current?.centerGraph(item);

Reply:
Gets passed an array, see:
https://github.com/reaviz/reagraph/blob/master/src/helpers/useSelection.ts#L156
```
ref.current?.centerGraph([data.id]);
```

Reply:
We could make this smarter so u don't have to pass an array if you think that would be helpful.

Reply:
I think it’s fine - supports the use cases where the graph gets centred on multiple nodes